File: UI/objectrelationshipui/src/app/models/Frame.ts
import { ObjectModel } from './ObjectModel';
import { TagModel } from './TagModel';

export interface Frame {
  id: number;
  position: { x: number; y: number };
  size: { w: number; h: number };
  origin: string;
  frameType: string;
  objectData?: ObjectModel[] ;
  tagData: TagModel[];
}

File: ObjectMetaDataTagging.Api/Models/Frame.cs
namespace ObjectMetaDataTagging.Api.Models
{
    public class Frame
    {
        public int Id { get; set; }
        public string Origin { get; set; }
        public ICollection<ObjectModel> ObjectData { get; set; }
        public ICollection<TagModel> TagData { get; set; }        

    }
}
